typescript/infra/scripts/agents/utils.ts

```diff
@@ -32,7 +32,8 @@ export class AgentCli {
     for (const role of this.roles) {
       switch (role) {
         case Role.Validator:
-          for (const chain of this.agentConfig.contextChainNames) {
+          for (const chain of this.agentConfig.contextChainNames[role]) {
+            if (chain !== 'solana' && chain !== 'nautilus') continue;
```

Comment:
If I remember correctly, my PR that was merged into this branch means this is no longer required?

Reply:
yes ty!